samples/bluetooth/hci_spi/src/main.c

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <stddef.h>
+#include <stdio.h>
+
+#include <zephyr.h>
+#include <misc/byteorder.h>
+#include <logging/sys_log.h>
+#include <misc/stack.h>
+
+#include <device.h>
+#include <init.h>
+#include <gpio.h>
+#include <spi.h>
+
+#include <net/buf.h>
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/l2cap.h>
+#include <bluetooth/hci.h>
+#include <bluetooth/buf.h>
+#include <bluetooth/hci_raw.h>
+#include <bluetooth/common/log.h>
```

Comment:
in the hci_uart we use #include "common/log.h", i tried that here for some reason cant include and compiler throws error.

The point is common/log.h is not public, not in include/bluetooth/ folder